<ui version="4.0" >
 <class>form_FileSend</class>
 <widget class="QDialog" name="form_FileSend" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Status</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>145</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QCheckBox" name="checkBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>FileInfo sended</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>FileTransfer Accepted</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Transfering</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Finished</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>266</x>
     <y>9</y>
     <width>254</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>FileInfo</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>FileName:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>FileSize:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>SizeTyp</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>240</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>abort</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>140</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>FileSend</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>progress</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" >
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>70</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
